addons/docs/src/blocks/DocsContext.ts

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+export interface DocsContextProps {
+  id?: string;
+  selectedKind?: string;
+  selectedStory?: string;
+  mdxKind?: string;
```

Comment:
What's this?

Reply:
Updated with comment explanation

Reply:
Is it because it is possible to render a different kind's stories in an mdx file?

Reply:
Yeah it has to do with that, but I don't remember the exact case 😺